packages/react-native/Libraries/StyleSheet/processBackgroundImage.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
```

Comment:
Please make sure this typechecks with `strict` mode, as Flow checks surprisingly little without it.

```suggestion
 * @flow strict-local
```